package.json

```diff
@@ -4,6 +4,7 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
+    "spellcheck": "find content -mindepth 1 -maxdepth 1 -type d | xargs -I {} bash -c 'cspell lint --no-progress --config {}/cspell.json {}/docs/pages/**/*.mdx'",
```

Comment:
Can we do without the `find`? Is yarn able to locate the cspell binaries when invoked via bash like this?

Something like:

```
cspell lint **/*.mdx
```

Reply:
By default, cspell looks for a configuration in the current directory. I wanted to make it possible for docs authors to edit the `cspell.json` in a particular `gravitational/teleport` submodule

Reply:
I'd just have one that is maintained at the docs directory or base dir . We already have a starting list of words

Reply:
It would be best if it's in the Teleport repo, as that's where the content is, and it's most efficient to be able to add new allowed words in the same PR that introduces them.

I don't see why we need a different config per-branch though, a single cpell.json in teleport/docs should do, right?

Reply:
By teleport/docs are you referring to the `gravitational/docs` repo?

`cspell.json` includes the [list of words to ignore](https://cspell.org/configuration/). If we expect a separate config file within each `gravitational/teleport` submodule, docs authors can add words to the list in the same PR where they introduce docs edits.

It's difficult to coordinate changes that involve both `gravitational/teleport` and `gravitational/docs` without stopping the docs from building for a few hours to a day or two. Since docs pages use a specialized vocabulary, new changes are likely to introduce unsupported words, and requiring PRs in two repos would place an undue burden on docs authors.

Reply:
No, I'm referring to the `docs` subdirectory of the `teleport` repo.

> and requiring PRs in two repos would place an undue burden on docs authors

That's exactly what I'm trying to avoid. I'm arguing for keeping the allow list int he same place as the content (teleport).

Reply:
Just to make sure I understand, how would the `spellcheck` command change if we were to put the `cspell.json` file in `gravitational/teleport/docs`? 

As far as I can tell, we would change it to:

```json
"spellcheck": "find content -mindepth 1 -maxdepth 1 -type d | xargs -I {} bash -c 'cspell lint --no-progress --config {}/docs/cspell.json {}/docs/pages/**/*.mdx'", 
```
I.e., we'd make the config path `{}/docs/cspell.json` instead of `{}/cspell.json`. 

This would be because `cspell.json` would be in the `docs` subdirectory of the `teleport` repo, which maps to each first-level subdirectory within `gravitational/docs/content` via git submodules. E.g., there would be a `cspell.json` in `content/12.x/docs`, `content/13.x/docs`, etc.

Reply:
Ah, right - that makes sense. Thanks for clarifying. This approach sounds good to me.